pkg/skaffold/runner/runner.go

```diff
@@ -66,9 +67,11 @@ type SkaffoldRunner struct {
 	test.Tester
 	tag.Tagger
 	sync.Syncer
-	watch.Watcher
+	filemon.Monitor
```

Comment:
Most of the embedded fields here shouldn't be exposed to the outside. I have it on my todo to fix my mistake. Could you make the monitor and the listener private?

Reply:
yeah that's a good point. I'll make these private and put a todo in there